relay/nakama/signer/kms.go

```diff
@@ -73,12 +81,16 @@
 // SignTx creates a sign.Transaction object with a signature. This doc page was used as a reference:
 // https://cloud.google.com/kms/docs/create-validate-signatures#validate_ec_signature
 func (k *kmsSigner) SignTx(ctx context.Context, personaTag string, namespace string, data any) (
+	*sign.Transaction, error) {
+	t, err := k.SignTxWithTimestamp(ctx, personaTag, namespace, data, time.Now().UnixMicro())
+	return t, err
+}
```

Comment:
From the documentation:

UnixMicro returns t as a Unix time, the number of microseconds elapsed since January 1, 1970 UTC. The result is undefined if the Unix time in microseconds cannot be represented by an int64 (a date before year -290307 or after year 294246). The result does not depend on the location associated with t.

See https://cs.opensource.google/go/go/+/refs/tags/go1.23.2:src/time/time.go;l=1259